edx_proctoring/api.py

```diff
@@ -1219,6 +1219,55 @@ def _get_email_template_paths(template_name, backend):
     return [base_template]
 
 
+def reset_practice_exam(exam_id, user_id):
+    """
+    Resets a completed practice exam attempt back to the created state.
+    """
+    log_msg = (
+        u'Resetting practice exam {exam_id} for user {user_id}'.format(
```

Comment:
#nit Explicit Unicode prefix looks redundant since we on a py3. 
And the question, do we aim to support py3.5? I'd like to start using f-strings ))), it looks like koa release would be on py3.8

Reply:
edx-platform still does support 3.5 at the moment so tentatively yes. I'll have to get back to you on when we can drop support in this library specifically.